packages/sdks/typescript/src/tests/helpers/chat.ts

```diff
@@ -8,27 +9,43 @@ import { vi } from 'vitest'
 type Server = ReturnType<typeof setupServer>
 const encoder = new TextEncoder()
 
-export function mockAuthHeader({
+export function mockRequest({
   server,
   apiVersion,
+  docPath,
 }: {
   server: Server
   apiVersion: SdkApiVersion
+  docPath: string
 }) {
-  const mockFn = vi.fn()
+  const mockAuthHeader = vi.fn()
+  const mockUrl = vi.fn()
+  const mockBody = vi.fn()
+  let body = {}
   server.use(
     http.post(
-      `http://localhost:8787/api/${apiVersion}/conversations/fake-document-log-uuid/chat`,
-      (info) => {
-        mockFn(info.request.headers.get('Authorization'))
+      `http://localhost:8787/api/${apiVersion}/conversations/${docPath}/chat`,
```

Comment:
docPath? this is the conversation uuid no?

Reply:
true